common/appointment.py

```diff
@@ -1,4 +1,13 @@
 import struct
```

Comment:
struct is not being used

Reply:
Removed from both here and `test/common/unit/test_appointment.py`.